db/schema.rb

```diff
@@ -79,6 +79,9 @@
     t.string   "reset_password_token"
     t.datetime "reset_password_token_expires_at"
     t.datetime "reset_password_email_sent_at"
+    t.string   "firstname"
```

Comment:
could you do a rollback `rake db:rollback` change `firstname` to `first_name` and same with last name and then call `rake db:migrate`? in rails it's recommended to use snake case for variables so let's do that :)